examples/elasticsearch_helper_example/src/Plugin/ElasticsearchIndex/SimpleNodeIndex.php

```diff
@@ -16,7 +20,42 @@
 class SimpleNodeIndex extends ElasticsearchIndexBase {
 
   /**
-   * NOTE: The structure of the indexed data is determined by normalizers,
-   * see NodeNormalizer.php.
+   * {@inheritdoc}
    */
+  public function getIndexDefinition() {
+    // Get field mappings.
+    $mappings = $this->getIndexMappings();
+
+    // Get index settings.
+    $settings = SettingsDefinition::create()
+      ->addOptions([
+        'number_of_shards' => 1,
+        'number_of_replicas' => 0,
+      ]);
+
+    $index_definition = IndexDefinition::create()
+      ->setMappings($mappings)
+      ->setSettings($settings);
+
+    // If you are using Elasticsearch < 7, add the type to the index definition.
+    $index_definition->setType($this->getTypeName([]));
+
+    return $index_definition;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getIndexMappings() {
+    $user_property = FieldDefinition::create('object')
+      ->addProperty('id', FieldDefinition::create('integer'))
```

Comment:
Should this be uid instead, to make it more familiar with Drupal properties and avoid confusion with the document id? Technically can be whatever.

Reply:
Good point. Changed from `id` to `uid` for better clarity.